### DISCUSSION TITLE
Add support for installation directory

### DISCUSSION THREAD

**Discussion by avioli** [milanvarady/Applite#34] [Ideas]
Brew supports specifying the casks `appdir` via an ENV variable, which unfortunately the app cannot read, since it is defined in my [shell rc-file](https://elv.sh/ref/command.html#rc-file).

```elvish
set-env HOMEBREW_CASK_OPTS --appdir=$E:HOME/Applications
```

or if defined in ZSH
```zsh
export HOMEBREW_CASK_OPTS="--appdir=$HOME/Applications"
```

It would be amazing if we could have a way to define that `appdir` within the app's settings.

Maybe as part of the onboarding setup and later available within the app's settings.

If you are open to PRs - I could give it a go.

Edit: apparently `brew install --cask --appdir=~/Applications ...` does the same. Some apps require to be in /Applications and can ignore the `--appdir` flag.

**Comment by milanvarady** [comment]
Support for `--appdir` has been added in the v1.2 release. You can select a custom installation directory during the setup, or in the settings.
